<HTML>
<HEAD>
<title>UnexpectedException.<init> - notesstub</title>
<link rel="stylesheet" href="..\..\..\style.css">
</HEAD>
<BODY>
<a href="../../index.html">notesstub</a>&nbsp;/&nbsp;<a href="../index.html">com.toolable.notes.stub.exception</a>&nbsp;/&nbsp;<a href="index.html">UnexpectedException</a>&nbsp;/&nbsp;<a href=".">&lt;init&gt;</a><br/>
<br/>
<h1>&lt;init&gt;</h1>
<code><span class="identifier">UnexpectedException</span><span class="symbol">(</span><span class="identifier" id="com.toolable.notes.stub.exception.UnexpectedException$<init>(kotlin.Throwable)/cause">cause</span><span class="symbol">:</span>&nbsp;<span class="identifier">Throwable</span><span class="symbol">)</span></code><br/>
<p>An exception that should never been thrown in theory, because the method ensure it.</p>
<p>Therefore, in practical cases, it allow to understand what gone wrong.</p>
<br/>
<br/>
<br/>
<br/>
</BODY>
</HTML>
